.circleci/scripts/git-diff-default-branch.ts

```diff
@@ -146,7 +148,7 @@ async function storeGitDiffOutputAndPrBody() {
 
     // Store the output of git diff
     const outputPath = path.resolve(CHANGED_FILES_DIR, 'changed-files.txt');
-    fs.writeFileSync(outputPath, diffOutput.trim());
+    fs.writeFileSync(outputPath, diffOutput?.trim() ?? '');
```

Comment:
should these change be made in `.github/scripts/git-diff-default-branch.ts` as well?